grails-forge-core/src/main/java/org/grails/forge/feature/build/gradle/templates/dependencies.rocker.raw

```diff
@@ -13,6 +13,7 @@ GradleBuild gradleBuild
 )
 
 dependencies {
+    implementation platform("org.grails:grails-bom:$grailsVersion")
```

Comment:
Isn't the bom already automatically added by grails-gradle-plugin?
How else are the versions currently resolving?

Reply:
This could now be removed after adding geb-spock and addressing buildSrc.